src/entropy-keys.ts

```diff
@@ -1,7 +1,12 @@
+import type { Eip1024EncryptedData, Hex } from '@metamask/utils';
+import { bytesToHex, concatBytes } from '@metamask/utils';
+import { utf8ToBytes } from '@noble/ciphers/utils';
+import { x25519 } from '@noble/curves/ed25519';
 import { secp256k1 } from '@noble/curves/secp256k1';
 import { sha256 } from '@noble/hashes/sha256';
 
 import { addressToBytes, bytesToAddress } from './utils/address-conversion';
+import { ERC1024 } from './utils/ERC1024';
```

Comment:
Can we rename this to EIP1024, did it ever move into the RC stage?

Reply:
It was never an EIP, as it didn't involve changes to the ethereum protocol.
Back in those days, people didn't complain about the difference between ERC/EIP but nowadays I've heard some complaints so I decided to use the "proper" name.

Reply:
Haha, fair enough!